test/src/test/java/com/distkv/dst/client/mastersalversync/TestMasterSalverSync.java

```diff
@@ -0,0 +1,92 @@
+package com.distkv.dst.client.mastersalversync;
+
+import com.distkv.dst.client.DefaultDstClient;
+import com.distkv.dst.client.DstClient;
+import com.distkv.dst.common.entity.sortedList.SortedListEntity;
+import com.distkv.dst.supplier.MasterSalverTestUtil;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+public class TestMasterSalverSync {
+
+  @Test
+  public void mainTest() throws InterruptedException {
+    MasterSalverTestUtil.startAllProcess();
+    TimeUnit.SECONDS.sleep(5);
+    DstClient client0 = new DefaultDstClient(String.format("list://127.0.0.1:%d", 8082));
+    DstClient client1 = new DefaultDstClient(String.format("list://127.0.0.1:%d", 8090));
+
+    //test
+    testStrPut(client0, client1);
+    testListPut(client0, client1);
+    testSetPut(client0, client1);
+    testDictPut(client0, client1);
+    testSlistPut(client0, client1);
+
+    MasterSalverTestUtil.stopAllProcess();
+    TimeUnit.SECONDS.sleep(1);
+  }
+
+  public void testStrPut(DstClient client0, DstClient client1) {
```

Comment:
add @Test annotation on the head of this method  please

Reply:
Same as above